debian/rules

```diff
@@ -13,6 +13,8 @@
 # package maintainers to append LDFLAGS
 #export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
+# configure with ULTs stack unwind support
+CONFIGURE_FLAGS += --enable-stack-unwind
```

Comment:
This is why it's a pity that `--enable-stack-unwind` isn't just implicit if the library is present.  You wouldn't have to add this if it were.